.travis.yml

```diff
@@ -3,7 +3,7 @@ language: scala
 env:
   global:
 
-script: sbt test:compile "hot -psource=scalap -w1 -f1" "micro/jmh:run -w1 -f1"
+script: sbt test:compile "hot -psource=scalap -w1 -f1" "; project compilation ; +test" "micro/jmh:run -w1 -f1"
```

Comment:
There's still a comment in the diff that links to https://github.com/lampepfl/dotty/issues/2704